src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -24,12 +26,115 @@ internal static class Globals
         internal const string LogTag = "Tizen.Sensor";
     }
 
+    /// <summary>
+    /// Data from sensor
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential, Pack = 0)]
+    public struct SensorEventStruct : IEquatable<SensorEventStruct>
+    {
+        /// <summary>
+        /// Accuracy of sensor data 
+        /// </summary>
+        public SensorDataAccuracy Accuracy
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// time stamp value 
+        /// </summary>
+        public UInt64 Timestamp
+        {
+            get;
+            private set;
+        }
+
+        /// <summary> 
+        /// value count
+        /// </summary>
+        public int ValueCount
+        {
+            get;
+            private set;
+        }
+
+        [MarshalAs(UnmanagedType.ByValArray, SizeConst = 16)]
+        private List<float> values;
+
+        /// <summary> 
+        /// sensor data values
+        /// </summary>
+        public List<float> Values
```

Comment:
X DO NOT use ArrayList or List<T> in public APIs.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections

Reply:
ArrayList replaced with arrays. done.